### ISSUE TITLE
Install KEDA for testing KEDA autoscaling for ConsumerGroup

### ISSUE THREAD

**Issue by pierDipi** [knative-extensions/eventing-kafka-broker#3237]
**Problem**

We should install KEDA for testing KEDA autoscaling for ConsumerGroup.

See initial POC: https://github.com/knative-sandbox/eventing-kafka-broker/pull/2796

**[Persona:](https://github.com/knative/eventing/blob/master/docs/personas.md)** 
Which persona is this feature for?
Contributors

**Exit Criteria**

- KEDA is installed when running `hack/deploy-infra.sh` (KEDA artifacts should checked in the repository, maybe in `third_party/keda`)
- KEDA can be updated with a script similar to `update-strimzi.sh`

**Time Estimate (optional):** 
How many developer-days do you think this may take to resolve?
1

**Comment by Cali0707** [comment]
@pierDipi does this depend on #2796?

**Comment by pierDipi** [comment]
> @pierDipi does this depend on #2796?

It's basically the same, but recorded into an issue

**Comment by Cali0707** [comment]
> > @pierDipi does this depend on #2796?
> 
> It's basically the same, but recorded into an issue

Okay, should we continue the work there or should I make a new PR?

**Comment by pierDipi** [comment]
my code is there and everyone can definitely take it as a starting point
